### PR TITLE
Require gegl-0.4 with gegl 0.4.14 support

### PR DESCRIPTION
**Pull request by rezso**
Updated version of https://github.com/mypaint/libmypaint/pull/135

### CODE REVIEW

**Comment by spixi** [comment]
This would be great. Gimp 2.10.8 requires GEGL >= 0.4.12, but libmypaint 1.3.0 requires GEGL = 0.3. I don't want to require two different GEGL instances on my system.

File: gegl/mypaint-gegl-surface.c [hunk 1]
```diff
@@ -78,7 +78,11 @@ tile_request_start(MyPaintTiledSurface *tiled_surface, MyPaintTileRequest *reque
 
     if (buffer_is_native(self)) {
         GeglBufferIterator *iterator = gegl_buffer_iterator_new(self->buffer, &tile_bbox, 0, self->format,
+#if GEGL_MAJOR_VERSION == 0 && GEGL_MINOR_VERSION == 4 && GEGL_MICRO_VERSION >= 14
+                                      read_write_flags, GEGL_ABYSS_NONE, 8);
```
Comment (hartwork) [inline gegl/mypaint-gegl-surface.c]: Why the magic 8 here?

**Comment by hartwork** [comment]
Any comments from the @mypaint team?

**Comment by jplloyd** [comment]
gegl-0.4 support added: 059e1993d472a578f2f90e4da906003a636549d4
Thank you for the patch, I changed it a bit to mark the API difference more clearly, and give a name to the magic number.

**Pull request closed without merging**